apps/ws/src/shared/subscriber-online/subscriber-online.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@nestjs/common';
+import { ISubscriberJwt } from '@novu/shared';
+import { SubscriberRepository } from '@novu/dal';
+
+@Injectable()
+export class SubscriberOnlineService {
+  constructor(private subscriberRepository: SubscriberRepository) {}
+
+  async handleConnection(subscriber: ISubscriberJwt) {
+    const isOnline = true;
+    const lastOnlineAt = new Date();
+    await this.updateOnlineStatus(subscriber, isOnline, lastOnlineAt);
```

Comment:
we actually don't have to set the `lastOnlineAt` on the new connection, only when the disconnect happens